packages/components/SelectionDropdown.tsx

```diff
@@ -0,0 +1,152 @@
+import React, { useEffect, useRef } from "react";
+import { TouchableOpacity, View, ViewStyle } from "react-native";
+
+import chevronDownSVG from "./../../assets/icons/chevron-down.svg";
+import chevronUpSVG from "./../../assets/icons/chevron-up.svg";
+import { BrandText } from "./BrandText";
+import { SVG } from "./SVG";
+import { Box } from "./boxes/Box";
+import { TertiaryBox } from "./boxes/TertiaryBox";
+import { useDropdowns } from "../context/DropdownsProvider";
+import {
+  neutral17,
+  neutral33,
+  neutral77,
+  neutralA3,
+  secondaryColor,
+} from "../utils/style/colors";
+import { fontSemibold13, fontSemibold14 } from "../utils/style/fonts";
+import { layout } from "../utils/style/layout";
+interface SelectionDropdownProps {
+  style?: ViewStyle;
+  onDropdownClosed?: () => void;
+  dropdownOptions: string[];
+  placeHolder?: string;
+  setItem: (item: string) => void;
+  item?: string;
+  label?: string;
+}
+
+export const SelectionDropdown = ({
+  style,
+  onDropdownClosed,
+  dropdownOptions,
+  placeHolder,
+  item,
+  label,
+  setItem,
+}: SelectionDropdownProps) => {
+  const { onPressDropdownButton, isDropdownOpen, closeOpenedDropdown } =
+    useDropdowns();
+  const dropdownRef = useRef<View>(null);
+
+  const isDropdownOpened = isDropdownOpen(dropdownRef);
+
+  useEffect(() => {
+    if (!isDropdownOpened) {
+      onDropdownClosed?.();
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Why we need this here ? , we can put `onDropdownClosed` in dependencies

Reply:
Okay, I will fix it.